src/content_scripts/profilePage.js

```diff
@@ -22,13 +22,16 @@ function createFollowButton() {
   username = username.split('#')[0];
   username = username.split('?')[0];
 
-  $(document).ready(function () {
-      if(window.location.href.indexOf("profile") > -1) {
-        $followButton = $('<button></button>').attr({ id: 'se_follow_button' }).css("margin-top", "6px");
-      } else {
-        $followButton = $('<button></button>').attr({ id: 'se_follow_button' }).css("bottom", "5px");
-      }
-  });
+  function makeStyledButton(profileElement, profileValue, generalElement, generalValue) {
+    if($('#tabs').find("a:first").hasClass("youarehere")) {
+      console.log($('#tabs').children("a:first").hasClass("youarehere"));
+      $followButton = $('<button></button>').attr({ id: 'se_follow_button' }).css(profileElement, profileValue);
+    } else {
+      $followButton = $('<button></button>').attr({ id: 'se_follow_button' }).css(generalElement, generalValue);
+    }
+  }
+
+  makeStyledButton("margin-top", "6px", "bottom", "5px");
```

Comment:
By looking at the code, I have couple of questions:

1. Earlier we were calling the method on document ready now we don't.
2. We were checking profile page specifically to insert different styling on different profile pages. Reason being there are so many stackexchange sites which follow a bit different url structure. 

Allow me sometime to carefully review this function change. I am sure you have tested this enough but still need to be sure.

Thanks!

Reply:
Thank you for fully reviewing the code. I hope it is not effusive or over-personal to say I am glad to see someone who is both polite and also rigorous.

Reasons for changes:

1) I tested with and without document ready and it *seemed* that document ready was not needed in this case, and while technically good practice, when implimented here resulted in slower load times, which was visually slightly jarring.

2) My opinion was that checking the URL was a hack, and on additional testing I noticed this did not work for some edge cases, such as when the page was manually reloaded, and once I figured out how to check it properly, I realised why my initial edit, while an improvement, felt "hacky".

A bit of background about me: I have about three years experience as an application developer and am grateful to work with someone who has significantly more experience and skill. Not only do I welcome a code review from a clearly highly competent coder, but I think it is imperative at my level that I am informed of best practice so that I can properly attain expert understanding of my profession.

Btw, have you considered implementing unit tests, or using something like Travis? That's a little over my head as I'm just a Software Developer, not a Senior, but I thought I'd mention just in case.

Reply:
My skype id is sachinjain024. I want to talk more about it.

Actually, this is one of the first projects I did when I started my career so code is not in best of its shape. And about Unit Testing and CI/CD with travis - YES I have done in many other projects.